engine/apps/api/serializers/alert_receive_channel.py

```diff
@@ -343,7 +341,7 @@ def _handle_core_template_updates(self, data, ret):
 
     def to_representation(self, obj):
         ret = super().to_representation(obj)
-        ret = self._get_templates_to_show(ret)
+        # ret = self._get_templates_to_show(ret)
```

Comment:
left comment

if we don't need it _get_templates_to_show function can be removed